lib/class-wp-rest-block-editor-settings-controller.php

```diff
@@ -75,7 +75,23 @@ public function get_items_permissions_check( $request ) {// phpcs:ignore Variabl
 	 * @return WP_Error|WP_REST_Response Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis
-		$editor_context = new WP_Block_Editor_Context();
+		switch ( $request['context'] ) {
+			case 'post-editor':
+			default:
```

Comment:
Shouldn't `default` be the last entry in `switch`?

Reply:
No, `default` and `case` statements can go anywhere. Putting it here ensures that `core/edit-post` is the default. `default` and `case` statements are essentially a `goto` with better marketing 😀